### ISSUE TITLE
Use of multicast instead of multiple unicast

### ISSUE THREAD

**Issue by groolot** [thomashollier/triptych#1]
You are defining your setup with RPi's that are in ethernet unicast, then you have to send your control message to each one instead of sending to a multicast IPv4 address that each one has subscribe to.

Your code should be lighter and your remote devices will receive the message exactly at the same time, even in a switched network.

**Comment by thomashollier** [comment]
Good idea! I'm a bit of a novice so implementing this is out of my immediate reach (I'll see if Google can give me the answers) but if anyone wants to take a crack at it, by all means...
